data/blog/tobin-harding-receives-lts-grant-for-rust-bitcoin.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: "Long-Term Support For Tobin Harding"
+date: '2023-12-21'
+tags: ['OpenSats', 'grants', 'bitcoin', 'LTS']
+draft: false
+authors: ['cfunk']
+images: ['/static/images/blog/19-tobin.jpg']
+summary: "OpenSats Welcomes Tobin Harding as an LTS Grantee."
+---
+
+OpenSats extends a warm welcome to Tobin Harding, our newest LTS grantee hailing
+from the Central Coast of New South Wales where he lives by the mantra
+*”Code, Roll, Dive, Breath - Repeat.”*
+
+In the heart of the kernel mini-conf at the Perth LCA nine years ago, Tobin felt
+a spark of inspiration as he overheard discussions about patching the kernel -
+"I want to do that." And the rest, as they say, is history.
+
+With the backing of this LTS grant, he is now set to keep achieving significant
+milestones for [rust-bitcoin](https://github.com/rust-bitcoin), crucial to
+projects like [Fedimint](https://github.com/fedimint/fedimint),
+[LDK](https://github.com/lightningdevkit/ldk-sample),
+[BDK](https://github.com/bitcoindevkit/bdk), and
+[electrs](https://github.com/romanz/electrs).
+
+Tobin's overarching goal is to release version 1.0.0 for
+[rust-secp256k1](https://github.com/rust-bitcoin/rust-secp256k1),
+[rust-bech32](https://github.com/rust-bitcoin/rust-bech32),
+[bitcoin_hashes](https://github.com/rust-bitcoin/bitcoin_hashes) and
+[rust-bitcoin](https://github.com/rust-bitcoin). He is also in the final stages
+of reviewing every public type and function, a meticulous process that often
+involves extensive discussions. This effort promises a stable API with minimal
+changes expected for at least a year.
```

Comment:
```suggestion
Tobin's overarching goal is to get `rust-bitcoin` to a version 1.0.0 release.
On the way, version 1.0.0 releases of 
[rust-secp256k1](https://github.com/rust-bitcoin/rust-secp256k1),
[rust-bech32](https://github.com/rust-bitcoin/rust-bech32),
[bitcoin_hashes](https://github.com/rust-bitcoin/bitcoin_hashes) and others.
He is also focused on the effort of reviewing every public type and function, a
meticulous process that often involves extensive discussions. This effort promises
a stable API with minimal changes that you guys can rely on.
```

Reply:
My whitespace may not be correct, may need changing to suit the rest of the document.